client/qiskit_serverless/core/clients/serverless_client.py

```diff
@@ -130,16 +130,27 @@ def __init__(  # pylint:  disable=too-many-positional-arguments
         try:
             channel_enum = Channel(channel)
         except ValueError as error:
-            raise QiskitServerlessException(
+            raise ValueError(
```

Comment:
Following @korgan00 feedback I just changed `QiskitServerlessException` by `ValueError` as a better way to manage this exception specifically.